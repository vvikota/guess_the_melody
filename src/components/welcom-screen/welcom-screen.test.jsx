import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from './welcom-screen';

it(`WelcomeScreen renders`, ()=> {
  const tree = renderer
    .create(<WelcomeScreen
      gameTime={5}
      maxMistakes={3}
      onClick={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
