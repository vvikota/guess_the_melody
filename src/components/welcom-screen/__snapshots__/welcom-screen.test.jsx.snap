// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen renders 1`] = `
<section
  className="welcome"
>
  <button
    className="welcome__button"
    onClick={[MockFunction]}
  >
    <span
      className="visually-hidden"
    >
      Начать игру
    </span>
  </button>
  <h2
    className="welcome__rules-title"
  >
    Правила игры
  </h2>
  <p
    className="welcome__text"
  >
    Правила просты:
  </p>
  <ul
    className="welcome__rules-list"
  >
    <li>
      За 
      5
       минут нужно ответить на все вопросы.
    </li>
    <li>
      Можно допустить 
      3
       ошибки.
    </li>
  </ul>
  <p
    className="welcome__text"
  >
    Удачи!
  </p>
</section>
`;
